index.css

```diff
@@ -13,57 +64,92 @@ body {
   margin: 0 auto;
 }
 
-ul {
-  list-style: none;
-  padding: 0;
-  margin: 0;
-}
-
-a {
-  text-decoration: none;
-  color: #adbac2;
+@media screen and (max-width: 1023px) {
+  .main {
+    padding: 0 10px;
+    margin: 0 auto;
+    }
 }
 
 /* HEADER */
 
+  .navigation {
+    width: 100%;
+  }
+
+@media screen and (max-width: 1023px) {
+  .navigation {
+    display: none;
+  }
+}
+
 .header-container {
-  padding: 30px 40px;
-  margin: 0 auto;
+  
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 60px;
+  width: 100%;
+  position: absolute;
+  padding: 0 10px;
+  background-color: var(--header-bg-color);
+  
+}
+
+@media screen and (max-width: 1023px) {
+  .header-container {
+    justify-content: flex-end;
+    height: 50px;
+
+  }
+}
+
+.burger-menu {
+  text-transform: uppercase;
+  display: none;
+}
+
+@media screen and (max-width: 1023px) {
+  .burger-menu {
+    display: block;
+  }
 }
 
 .navigation-list {
   display: flex;
-  flex-direction: row;
+
   align-items: center;
   justify-content: space-between;
+  width: 100%;
+  padding: 0 10px;
 }
 
 .nav-item-link {
   line-height: 1;
   letter-spacing: 0.03em;
 
   font-size: 20px;
+  white-space: nowrap;
 }
 
 .nav-item-link:hover,
 .nav-item-link:focus {
-  color: rgb(67, 155, 245);
+  color: var(--accent-color);
 }
 
 /* HERO */
 
-img {
-  display: block;
-  max-width: 100%;
-  height: auto;
+@media screen and (max-width: 1023px) {
+  .hero-text {
+    font-weight: 400;
+    font-size: 16px;
+  }
 }
 
+
 .hero {
   padding: 250px 40px 70px 40px;
-  background-image: linear-gradient(
-      rgba(47, 48, 58, 0.4),
-      rgba(47, 48, 58, 0.4)
-    ),
+  background-image: var(--hero-bg-img-color),
```

Comment:
Even if the images are opaque and the color won't be displayed in normal circumstances, web developers should always specify a [background-color](https://developer.mozilla.org/en-US/docs/Web/CSS/background-color). If the images cannot be loaded—for instance, when the network is down—the background color will be used as a fallback.

Reply:
just add bg color

Reply:
?